wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -36,6 +36,15 @@ <h1>
         <a href="{% url 'wafer_user_profile' username=author.username %}">{{ author.userprofile.display_name }}</a>
       {% endfor %}
     </p>
+    {% if user.is_staff or perms.talks.view_all_talks %}
+      {% for author in object.authors.all %}
+      <p class='bio'>
```

Comment:
The general style we've used is to indent both for HTML and Django template blocks.

Reply:
fixed